torchtune/models/gemma2/_component_builders.py

```diff
@@ -115,8 +114,7 @@ def gemma2(
     """
     rope = RotaryPositionalEmbeddings(dim=head_dim, max_seq_len=max_seq_len, base=rope_base)
     
-    layers = torch.nn.ModuleList()
-    
+    layers = nn.ModuleList()
```

Comment:
super nit: let's keep `torch.nn.ModuleList()` as is, there was some big PR a long time ago to keep torch.nn

Reply:
Yeah looks like we aren't even using `nn.ModuleList` vs `torch.nn.ModuleList` consistently today. If you want to standardize on `torch.nn.ModuleList` that's great but not a blocker for this PR